README.md

```diff
@@ -1,4 +1,6 @@
-[![Build Status](https://travis-ci.org/elifesciences/journal-cms.svg?branch=develop)](https://travis-ci.org/elifesciences/journal-cms)
+# eLife Journal CMS
+
+[![Build Status](http://ci--alfred.elifesciences.org/buildStatus/icon?job=test-journal-cms)](http://ci--alfred.elifesciences.org/job/test-journal-cms/)
```

Comment:
This is unrelated but needs to be updated as travis-ci hasn't been hooked up with this repo for a long time!